btr-web/btr-main-app/pages/requestToOmit.vue

```diff
@@ -13,58 +28,88 @@ const { completingPartyRef, omitObscureRef, siBizRef } = storeToRefs(omitIndivid
         <h3 class="text-xl font-bold mb-2.5">
           {{ $t('general.regName') }}
         </h3>
-        <h1 class="text-2xl font-bold mb-4" data-cy="request-to-omit-title">
-          {{ $t('labels.requestOmit.header') }}
-        </h1>
-        <p class="text-base mb-4" data-cy="request-to-omit-text">
-          {{ $t('texts.requestOmitText') }}
-        </p>
-        <BcrosHelpTip
-          :title="$t('helpTitles.requestOmit.closed')"
-          :title-expanded="$t('helpTitles.requestOmit.expanded')"
-        >
-          <template #default>
-            <BcrosI18HelperLink translation-path="helpTexts.requestOmit.detail" />
-          </template>
-        </BcrosHelpTip>
+        <span v-if="showSubmission">
+          <h1 class="text-2xl font-bold mb-4" data-cy="request-to-omit-title">
+            {{ $t('labels.requestOmit.header') }}
+          </h1>
```

Comment:
It is better to use `<template v-if="">` so we don't add block-level elements inside inline element